modules/nw-networkpolicy-audit-configure.adoc

```diff
@@ -120,12 +120,20 @@ kind: Pod
 metadata:
   name: client
 spec:
+  securityContext:
```

Comment:
Thanks @asood-rh - option 1 sounds better to me too. 

It looks like we talk about working in the `default` namespace in lines 113, 117, 185, and 189.

Can I just change all of those instances to some other example namespace (`test-namespace` or something like that), or is there more adjustment than that needed?

Reply:
Yes @bergerhoffer You could use namespace like "test-namespace" or "client-namespace" instead of 'default" and interchange it with references to 'default'.

Reply:
@asood-rh Updated, let me know how it looks now!

Reply:
/lgtm